qutip/entropy.py

```diff
@@ -223,45 +224,79 @@ def entropy_mutual(rho, selA, selB, base=e, sparse=False):
     return out
 
 
-def _entropy_relative(rho, sigma, base=e, sparse=False):
+def entropy_relative(rho, sigma, base=e, sparse=False, tol=1e-12):
     """
-    ****NEEDS TO BE WORKED ON****
-
     Calculates the relative entropy S(rho||sigma) between two density
     matrices.
-
     Parameters
     ----------
```

Comment:
There's a missing blank line for the section heading here.

Reply:
The robots caught that too -- woot.

Reply:
Oh, that's great to see that they're working like that.  Let me see if I can force coloured output of Sphinx, though - might make it a bit easier to pick out the failures.